app/controllers/api/services_controller.rb

```diff
@@ -18,6 +18,20 @@ def edit_resource(type, id, data)
       super(type, id, attributes)
     end
 
+    def add_resource_resource(type, id, data)
+      svc = resource_search(id, type, collection_class(type))
+      data['resources'].each do |resource_ref|
+        resource_type, resource_id = parse_href(resource_ref['href'])
+        resource = resource_search(resource_id, resource_type, collection_class(resource_type))
```

Comment:
@jntullo 
Should we validate if the resource can be found, since we carry hrefs if the object is deleted in the meantime we won't have a resource?